### PR TITLE
Events cronjob schedule

### PR DESCRIPTION
**Pull request by NoahNsimbe**
# Adjusting events check cronjob

**_WHAT DOES THIS PR DO?_**

- [x] Removes duplicated yaml files
- [x] Adjusts schedule for the job that checks whether hourly and raw measurements exist from 0 to 30 minutes pasted the hour

**_WHAT JIRA STORY/TASK IS RELATED TO THIS PR?_**
None

**_HOW DO I TEST OUT THIS PR?_**

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**

N/A

**_ARE THERE ANY RELATED PRs?_**

N/A

### CODE REVIEW

File: k8s/events-monitor/prod-events-notifs-cronjob.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ metadata:
 spec:
   successfulJobsHistoryLimit: 1
   failedJobsHistoryLimit: 1
-  schedule: "@hourly"
+  schedule: "30 * * * *"
```
Comment (NoahNsimbe) [inline k8s/events-monitor/prod-events-notifs-cronjob.yaml:9]: Changed the schedule to 30 minutes pasted the hour. @Baalmart

File: k8s/events-monitor/prod-cronjob.yaml [hunk 2]
```diff
@@ -1,36 +0,0 @@
-apiVersion: batch/v1beta1
```
Comment (NoahNsimbe) [inline k8s/events-monitor/prod-cronjob.yaml:1]: Deleted this file because it's a duplicate. @123MwanjeMike

**Review by 123MwanjeMike — APPROVED** [review]

File: k8s/events-monitor/prod-events-notifs-cronjob.yaml (hunk 1 above)
Comment (Baalmart) [reply k8s/events-monitor/prod-events-notifs-cronjob.yaml:9]: Awesome, thanks @NoahNsimbe

**Review by Baalmart — APPROVED** [review]
LGTM

**Pull request merged**